package.json

```diff
@@ -1,28 +1,70 @@
 {
-  "name": "web-ui",
+  "name": "stockmarket-simulation",
   "version": "0.1.0",
   "homepage": ".",
   "private": true,
+  "jest": {
+    "collectCoverageFrom": [
+      "src/**/*.{ts,tsx}",
+      "!<rootDir>/node_modules/",
+      "!src/**/*.story.{js,jsx,ts,tsx}",
+      "!src/**/index.{ts,tsx}",
+      "!src/**/*Mocks.{ts,tsx}",
+      "!src/**/*Selectors.{ts,tsx}",
+      "!src/**/testUtils.{ts,tsx}",
+      "!src/registerServiceWorker.ts"
+    ],
+    "coverageThreshold": {
+      "global": {
+        "branches": 0,
+        "functions": 0,
+        "lines": 0,
+        "statements": 0
+      }
+    }
+  },
   "dependencies": {
+    "enzyme": "^3.2.0",
+    "json-loader": "^0.5.7",
+    "moment": "^2.22.1",
```

Comment:
Why not date-fns?
https://github.com/date-fns/date-fns/issues/275#issuecomment-264934189

Reply:
But we can also use moment. I don't care too much.

Reply:
I think momentjs is easier to use

Reply:
Not much difference between them. But let's use Moment then.